app/src/userpages/components/KeyField/index.jsx

```diff
@@ -179,7 +171,7 @@ const KeyField = ({
                     </div>
                     <WithInputActions actions={inputActions}>
                         <Text
-                            value={displayValue}
+                            value={value && truncate(value)}
```

Comment:
If `value` isn't an address then it'll return raw `value`, meaning we should be just fine going with just `truncate(value)`.

Reply:
No pressure. This here works just fine.

Reply:
Yep, it would be fine otherwise but flow is complaining... I was too lazy to remove flow from this file

Reply:
Got it!